js/main.js

```diff
@@ -1,2 +1,27 @@
 /* global data */
 /* exported data */
+var $form = document.querySelector('form');
+var $img = document.querySelector('img');
+
+$form.addEventListener('focusout', function (event) {
+  if (event.target.matches('.photo-url')) {
+    $img.setAttribute('src', event.target.value);
+  }
+  if (event.target.matches('.photo-url') && event.target.value === '') {
+    $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  }
+});
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entry = {
+    title: $form.title.value,
```

Comment:
You need to set an `entryId` using `data.nextEntryId`

Reply:
<img width="1509" alt="Screen Shot 2021-09-08 at 12 53 05 PM" src="https://user-images.githubusercontent.com/85265067/132576605-7f4101bb-0bee-4233-8a81-952278ad513c.png">